src/org/daisy/dotify/formatter/impl/page/PageImpl.java

```diff
@@ -113,6 +115,7 @@ void newRow(RowImpl r) {
 			if (!topPageAreaProcessed) {
 				addTopPageArea();
 				getDetails().startsContentMarkers();
+				startsContentIdentifiers();
```

Comment:
Please put the `contentIdentifiersBegin` variable and logic in the `PageDetails` class
so that the statement here will be: `getDetails().startsContentIdentifiers()`

Reply:
OK that kinda makes sense.

Reply:
Fixed